src/tools/constants.rs

```diff
@@ -1,7 +1,7 @@
 use std::time::Duration;
 
 /// Timeout when waiting for [Node](crate::setup::node::Node)'s start.
-pub const CONNECTION_TIMEOUT: Duration = Duration::from_secs(30);
```

Comment:
Sorry about this. Still, my tests sometimes fail on my Mac.

Reply:
Undone. The real fix is on this line.
https://github.com/runziggurat/xrpl/pull/44/files#diff-9fb30675c3f486d6795fcb124619aadc8a7bef02685b2d39275707e7ce9a52edR38